README.md

```diff
@@ -389,6 +389,7 @@ These settings can be changed at any time.
   Bottlerocket sets this value to false by default. 
 * `settings.ecs.loglevel`: The level of verbosity for the ECS agent's logs.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
+* `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
```

Comment:
```suggestion
* `settings.ecs.enable-spot-instance-draining`: If the instance receives a [Spot](https://aws.amazon.com/ec2/spot/) termination notice, the agent will set the instance's state in ECS to `DRAINING` so the workload can be moved gracefully.  Defaults to `false`.  (The instance will need to have permission to call the UpdateContainerInstancesState API, per [ECS documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-instance-draining.html).)
```
I think we should mention the permission that the instance has to have, so the user knows what they have to set up beforehand.